saskatoon/sitebase/templates/app/list_views/filters.html

```diff
@@ -18,7 +18,6 @@ <h4 class="panel-title">
             <div class="collapse" id="accordionGreen-one" role="tabpanel">
                 <div class="panel-body" style="display:inline-block;">
                     <form role="form">
-                        {% csrf_token %}
```

Comment:
Why this change exactly ?

Reply:
As far as I know a crsf token is not needed for a GET request and it's really annoying to have it in the url while testing filters